libc/test/src/math/CMakeLists.txt

```
@@ -2368,6 +2368,37 @@ add_fp_unittest(
     libc.src.stdlib.srand
 )
 
+add_fp_unittest(
+  dfmal_test
+  NEED_MPFR
+  SUITE
+    libc-math-unittests
+  SRCS
+    dfmal_test.cpp
+  HDRS
+    FmaTest.h
+  DEPENDS
+    libc.src.math.dfmal
+    libc.src.stdlib.rand
+    libc.src.stdlib.srand
+)
+
+add_fp_unittest(
+  dsubl_test
+  NEED_MPFR
+  SUITE
+    libc-math-unittests
+  SRCS
+    dsubl_test.cpp
+  HDRS
+    SubTest.h
+  DEPENDS 
+    libc.src.errno.errno
+    libc.hdr.fenv_macros
+    libc.src.math.dsubl
```

libc/test/src/math/SubTest.h doesn't use `errno` or `fenv_macros`: https://github.com/llvm/llvm-project/blob/ff8a27f7393d0441aa220d0ecb3e1c7485c4449b/libc/test/src/math/SubTest.h#L12-L15

```suggestion
  DEPENDS
    libc.src.math.dsubl
```